### PR TITLE
docs(tutorials): review content

### PR DESCRIPTION
**Pull request by bene2k1**
### Description
Second part of tutorials review for MTA-2390

### CODE REVIEW

File: tutorials/installation-jekyll/index.mdx [hunk 1]
```diff
@@ -506,4 +506,4 @@ Jekyll uses the [Liquid](https://shopify.github.io/liquid/) templating language
 
 More information on the directory structure of Jeykill is available directly in their [documentation](https://jekyllrb.com/docs/structure/).
 
-In this tutorial you have installed a Jekyll development Instance to deploy your website and to run it locally on this machine for testing purposes. We have also set up Git to push the website to a production server and to generate it automatically with Jekyll, before it is finally served to our users by Nginx. If you want to learn more about Jekyll, you can read the [official documentation](https://jekyllrb.com/docs/home/).
+In this tutorial you have installed a Jekyll development Instance to deploy your website and to run it locally on this machine for testing purposes. You have also set up Git to push the website to a production server and to generate it automatically with Jekyll, before it is finally served to our users by Nginx. If you want to learn more about Jekyll, you can read the [official documentation](https://jekyllrb.com/docs/home/).
```
Comment (farhakennar) [inline tutorials/installation-jekyll/index.mdx:509]:
```suggestion
In this tutorial you have installed a Jekyll development Instance to deploy your website and run it locally on this machine for testing purposes. You have also set up Git to push the website to a production server and generate it automatically with Jekyll, before it is finally served to our users by Nginx. If you want to learn more about Jekyll, read the [official documentation](https://jekyllrb.com/docs/home/).
```

**Review by farhakennar — APPROVED** [review]

File: tutorials/jitsi-docker/index.mdx [hunk 2]
```diff
@@ -20,7 +20,7 @@ This tutorial explains how to install the Jitsi Meet solution on a virtual cloud
 <Message type="requirement">
   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
   - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key)
-  - You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Bionic Beaver
+  - You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Bionic Beaver or later
   - You have installed [Docker](docs/tutorials/install-docker-ubuntu-bionic) on the instance
```
Comment (nedaehm) [inline tutorials/jitsi-docker/index.mdx:24]:
```suggestion
  - You have installed [Docker](docs/tutorials/install-docker-ubuntu-bionic) on the Instance
```

File: tutorials/jitsi-docker/index.mdx [hunk 3]
```diff
@@ -20,7 +20,7 @@ This tutorial explains how to install the Jitsi Meet solution on a virtual cloud
...
   - For best performances of Jitsi Meet, an instance with at least 4GB RAM is recommended
```
Comment (nedaehm) [inline tutorials/jitsi-docker/index.mdx:25]:
```suggestion
  - For best performances of Jitsi Meet, we recommend you use an Instance with at least 4GB RAM
```

File: tutorials/lb-firewall-haproxy-pfsense/index.mdx [hunk 4]
```diff
@@ -1,9 +1,9 @@
 ---
 meta:
-  title: How to Build a Redundant Loadbalancer and Firewall Infrastructure With pfSense and HAproxy on a Scaleway Dedibox
+  title: Building a redundant load balancer and firewall infrastructure with pfSense and HAproxy on a Scaleway Dedibox
```
[the remark this answers is not in the document]
Comment (bene2k1) [reply tutorials/lb-firewall-haproxy-pfsense/index.mdx:3]: not a product name here :)

File: tutorials/lb-firewall-haproxy-pfsense/index.mdx [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
   description: This page explains how to install pfSense on a Scaleway Dedibox dedicated server
 content:
-  h1: How to Build a Redundant Loadbalancer and Firewall Infrastructure With pfSense and HAproxy on a Scaleway Dedibox
+  h1:Building a redundant load balancer and firewall infrastructure with pfSense and HAproxy on a Scaleway Dedibox
```
[the remark this answers is not in the document]
Comment (bene2k1) [reply tutorials/lb-firewall-haproxy-pfsense/index.mdx:6]: not a product name here :)

**Review by nedaehm — APPROVED** [review]

**Pull request merged**